pycord/ui/house.py

```diff
@@ -75,7 +74,9 @@ def add_component(self, comp: Component) -> None:
             The component to append
         """
         if len(self.components) == 5:
-            raise ComponentException('Cannot add more components, already reached maximum')
+            raise ComponentException(
+                'Cannot add more components, already reached maximum'
+            )
```

Comment:
Reverts out of scope change.
```suggestion
            raise ComponentException('Cannot add more components, already reached maximum')
```